src/zmalloc.h

```diff
@@ -47,6 +48,7 @@
 
 #elif defined(USE_JEMALLOC)
 #define ZMALLOC_LIB ("jemalloc-" __xstr(JEMALLOC_VERSION_MAJOR) "." __xstr(JEMALLOC_VERSION_MINOR) "." __xstr(JEMALLOC_VERSION_BUGFIX))
+#define defrag_supported "yes"
```

Comment:
I would rather simply compare the allocator, if it's JEMALLOC print yes else no for now. Won't require us to add another macro.